### PR TITLE
add prefers-reduced-motion to accessibility docs

### LINKED ISSUE #11890: Add Accessibility section on prefers-reduced-motion
### Pertinent section of the Wagtail docs

https://docs.wagtail.org/en/stable/advanced_topics/accessibility_considerations.html#content-modeling

### Details

It would be good for the Accessibility page to have a section on prefers-reduced-motion.

Respect for prefers-reduced-motion is not required by WCAG 2.0-2.2, but is a best practice. If the website visitor or editor has set their OS to indicate they don't want animation, the site doesn't serve them cosmetic animation. Supported by Chrome, Safari, Firefox.

Note: The word "reduced" is an artifact of Apple using the terminology "reduce motion" in iOS and macOS before other operating systems implemented this feature. In Windows and Android the settings terminology implies all or nothing, but is actually the same signal to browsers.

### PR DESCRIPTION
**Pull request by RoelKoper**
add documentation on prefers-reduced-motion in the accessibility docs.
Fixes #11890

### CODE REVIEW

**Review by stevejalim — APPROVED** [review]
Thanks for this - as someone who personally needs to 'prefer reduced motion', it's always good to see this behaviour surfaced as an option

File: docs/advanced_topics/accessibility_considerations.md [hunk 1]
```diff
@@ -224,6 +224,22 @@ Readability is fundamental to accessibility. One of the ways to improve text con
 
 (accessibility_resources)=
 
+### prefers-reduced-motion
+
+Some users may prefer to see a more static version of your site, for example, users with vestibular disorders. You can respect this preference by using the `prefers-reduced-motion` media query in your CSS.
```
Comment (zerolab) [inline docs/advanced_topics/accessibility_considerations.md:229]: A small suggestion for the sentence flow:

```suggestion
Some users, such as those with vestibular disorders, may prefer a more static version of your site. You can respect this preference by using the `prefers-reduced-motion` media query in your CSS.

```

File: docs/advanced_topics/accessibility_considerations.md [hunk 2]
```diff
@@ -224,6 +224,22 @@ Readability is fundamental to accessibility. One of the ways to improve text con
...
+@media (prefers-reduced-motion) {
+  /* styles to apply if a user's device settings are set to reduced motion */
+    /* for example, disable animations */
+    * {
+        animation: none !important;
+        transition: none !important;
+    }
+}
+```
+Note that `prefers-reduced-motion` is only applied to users with this setting enabled in their operating system or browser. This feature is supported by Chrome, Safari and Firefox. For more information on reduced motion, see the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).
```
Comment (zerolab) [inline docs/advanced_topics/accessibility_considerations.md:240]:
```suggestion

Note that `prefers-reduced-motion` is only applied for users who enabled this setting in their operating system or browser. This feature is supported by Chrome, Safari and Firefox. For more information on reduced motion, see the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).
```

**Review by zerolab — CHANGES_REQUESTED** [review]
Made a couple of minor suggestions to tweak the sentence flow, and add some spacing around the CSS code block

Thank you for contributing this @RoelKoper

**Review by zerolab — APPROVED** [review]

**Comment by zerolab** [comment]
Merged in 366a7b7.
Thank you for your first contribution to Wagtail @RoelKoper 👏🏼

**Pull request closed without merging**